charts/capsule/Chart.yaml

```diff
@@ -21,7 +21,7 @@ sources:
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.0.19
+version: 0.0.20
```

Comment:
There's no need for this version bump since we're going to release a huge release with all the changes so far.